index.js

```diff
@@ -163,7 +163,7 @@ program
       })
       .then(() => {
         const watcher = chokidar.watch(rootFolder, {
-          ignored: [outputFolder, /(^|[/\\])\../, '/**/*.md__jb_tmp__'],
+          ignored: [outputFolder, /(^|[/\\])\../, '*.md__jb_tmp__', '*.html__jb_tmp__'],
```

Comment:
I'm really sorry, the previous suggestion that i mentioned didn't work - first time i tested i did not have any temp files built..
![image](https://user-images.githubusercontent.com/22221132/37198268-bbcf0cac-23b8-11e8-8ba0-d05f542e8397.png)

```js
ignored: [outputFolder, /(^|[/\\])\../, path => path.endsWith('___jb_tmp___')],
```

I've tested this multiple times and had no issue. Would this be better?